### PR TITLE
INV-29605: Updating Documentation for new Field in API Response

### PR DESCRIPTION
**Pull request by bboddu**

### CODE REVIEW

**Comment by bboddu** [comment]
@sarraloew Please review and approve

**Review by sarraloew — APPROVED** [review]
Looks good, thanks.

File: src/api-reference/invoice/payment-request.markdown [hunk 1]
```diff
@@ -146,6 +147,12 @@ Name | Type | Format | Description
 `Custom11` through `Custom20`	|	`string`	|	-	|	The details from the Custom fields. These may not have data, depending on configuration.
 `Percentage`	|	`string`	|	-	|	**Required** The percentage of the Request Line Item that the individual allocation record. All Allocations associated to a given Line Item should add up to 100.
 
+### MatchedPurchaseOrderReceipt
+
+Name | Type | Format | Description
```
Comment (charlieconcur) [inline src/api-reference/invoice/payment-request.markdown:152]: This Markdown needs to be brought into alignment with the documentation template located at https://github.com/concur/developer.concur.com/blob/preview/src/document-template.markdown.

**Review by charlieconcur — DISMISSED** [review]

**Comment by bboddu** [comment]
@charlieconcur I added missing links to objects and array. Please review and update me if any other changes are needed

**Pull request merged**